### ISSUE TITLE
Improve PWA uikit test website features

### ISSUE THREAD

**Issue by adamzerella** [govau/uikit#543]
It seems that the current uikit service isn't very progressive, there's a few things we can look at:

- [ ] Add `/robots.txt` to production environment, this was added for staging/testing with the recent `nginx_buildpack` addition.
- [ ] Add `/sitemap.xml` this is great for SEO and users looking for a perfect reference of site pages.
- [ ] Add service worker to improve asset caching and offline access
- [ ] Missing `/manifest.json`
- [ ] Missing `<meta description=""/>` element
- [ ] Missing `<meta theme-color=""/>` element
- [ ] Poor viewport optimization

This issue should be separate into an epic or individual tasks at the least.

**Comment by rhubarbselleven** [comment]
Wouldn't this be supportive documentation? Typically these features would be provided by the CMS being used.

As an implementer I would not expect this as a feature (but happy to take the recommendations).

**Comment by alex-page** [comment]
I am going to close this issue, the test website is not meant to be user facing or have good SEO. This would be a waste of time IMO.

@adamzerella let me know if you disagree, happy to talk about it. Maybe some of these things we can implement, however I don't think they add much value.
